pagopa/io-app
# feat: [#176441178] Show the bottom navigator only on first-level routes

## How to test
0. Login and land on the "Messages" tab
1. Go to e.g. "Profile", you should still see the TabBar
2. Navigate to any sub-route, you should no longer see the bottom bar
